logic/src/androidAndroidTest/kotlin/com/wire/kalium/logic/feature/call/scenario/OnCloseCallTest.kt

```diff
@@ -73,19 +83,70 @@ class OnCloseCallTest {
         // given
         // when
         onCloseCall.onClosedCall(
-            reason = 0,
-            conversationId = "conversationId@domainId",
+            reason = 4,
+            conversationId = conversationIdString,
             messageTime = Uint32_t(value = 1),
-            userId = "userId@domainId",
-            clientId = "clientId",
+            userId = userId,
+            clientId = clientId,
             arg = null
         )
         advanceUntilIdle()
 
         // then
         verify(callRepository)
             .suspendFunction(callRepository::updateCallStatusById)
-            .with(eq("conversationId@domainId"), eq(CallStatus.CLOSED))
+            .with(eq(conversationIdString), eq(CallStatus.MISSED))
+            .wasInvoked(once)
+
+        verify(callRepository)
```

Comment:
is that correct? since the conversation has ongoing call, why we need to persist the missedCall?

Reply:
Oh! it should not be  `verify(callRepository)`...In this case, ongoing in other platforms but it's closed in the current client